ui/lib/utils/reportData.ts

```diff
@@ -0,0 +1,17 @@
+import mixpanel from 'mixpanel-browser'
+import client from '@lib/client'
+
+export async function init() {
+  mixpanel.init('cb1135f29a413c653332990f07b3586a', {
+    opt_out_tracking_by_default: true,
+  })
+  // check option
+  const res = await client.getInstance().getInfo()
+  if (res?.data?.enable_report) {
+    mixpanel.opt_in_tracking()
+  }
+}
+
+export function report(eventType: string, eventBody: object) {
+  mixpanel.track(eventType, eventBody)
+}
```

Comment:
I would prefer to not encapsulate the API so that we are free to use all Mixpanel APIs.

Reply:
So we will always use MixPanel, won't change it later?

Reply:
Nop, we definitely would change. But I think current abstraction is not enough for a painless vendor change.